aiodocker/jsonstream.py

```diff
@@ -18,16 +23,20 @@ def __init__(self, response, transform=None):
                 data = await self.response.content.readline()
                 if not data:
                     break
-            except (aiohttp.ClientConnectionError,
-                    aiohttp.ServerDisconnectedError):
+            #except (errors.ClientDisconnectedError,
+            #        errors.ServerDisconnectedError):
+            except errors.ServerDisconnectedError:
                 break
             yield self.transform(json.loads(data.decode('utf8')))
 
     async def close(self):
         # response.release() indefinitely hangs because the server is sending
         # an infinite stream of messages.
         # (see https://github.com/KeepSafe/aiohttp/issues/739)
-        await self.response.close()
+
+        # response error , it has been closed
+        if self.response.close():
```

Comment:
read aiohttp client Response code ,,,   response.close  is not async ,  now remove 'await' to fix it